src/lib/containers/table/table-top/ProgrammaticTableDownload.tsx

```diff
@@ -45,40 +45,70 @@ function ProgrammaticOptions({
         // SWC-5686: The ID column is required by the client, and this column may not have been selected!
         // Change the SQL to "SELECT * ..."
         const indexOfFrom = res.transformedSql.toUpperCase().indexOf('FROM SYN')
-        const selectStarTransformedSql = `SELECT * ${res.transformedSql.substring(indexOfFrom)}`
-        setGeneratedSql(selectStarTransformedSql.replace(/"/g, '\\"'))
+        const selectStarTransformedSql = `SELECT * ${res.transformedSql.substring(
+          indexOfFrom,
+        )}`
+        setGeneratedSql(selectStarTransformedSql)
       } catch (e) {
         console.error('Error on getTransformSqlWithFacetsRequest ', e)
       }
     }
     getData()
   }, [queryBundleRequest, queryResultBundle])
 
+  // Replace quotation marks with escaped quotations. For CLI, also escape backticks.
+  const commandLineSql = useMemo(
+    () => generatedSql.replace(/(["`])/g, '\\$1'),
+    [generatedSql],
+  )
```

Comment:
This is the main logic change. For command line instructions, also escape backticks.